### PR TITLE
Update issue template adding more links

### PR DESCRIPTION
**Pull request by ringabout**
After this PR

image

### CODE REVIEW

**Comment by juancarlospaco** [comment]
I do not agree, not everybody uses Discord, maybe try to replicate the web:

temp

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - name: Forum
+    url: https://forum.nim-lang.org
+    about: Please ask and answer questions here.
+  - name: Discord
+    url: https://discord.com/invite/nim
+    about: Please ask and answer questions here.
```
Comment (ringabout) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
```

Comment (timotheecour) [reply .github/ISSUE_TEMPLATE/config.yml]: actually, I think it's good to mention discord. User may be missing out on what happens there.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - name: Forum
+    url: https://forum.nim-lang.org
+    about: Please ask and answer questions here.
+  - name: Discord
+    url: https://discord.com/invite/nim
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://discord.gg/nim
```

Shorter link :)

**Review by timotheecour — APPROVED** [review]

**Pull request merged**